src/test/groovy/org/prebid/server/functional/tests/TimeoutSpec.groovy

```diff
@@ -136,12 +145,14 @@ class TimeoutSpec extends BaseSpec {
         and: "Pbs config with default request"
         def pbsContainer = new PrebidServerContainer(
                 ["default-request.file.path" : APP_WORKDIR + defaultRequest.fileName,
-                 "auction.max-timeout-ms"    : MAX_TIMEOUT as String,
-                 "auction.default-timeout-ms": DEFAULT_TIMEOUT as String]).tap {
+                 "auction.biddertmax.max"    : MAX_TIMEOUT as String]).tap {
             withCopyFileToContainer(MountableFile.forHostPath(defaultRequest), APP_WORKDIR) }
         pbsContainer.start()
         def pbsService = new PrebidServerService(pbsContainer)
 
+        and: "Idle request PBS for stabilize timeout"
+        pbsService.sendAuctionRequest(BidRequest.defaultBidRequest)
```

Comment:
Please move this logic to the `PrebidServerService`. Create a method that will warmup the container on-demand. Something like `PrebidServerService withWarmup()` so that you can call it fluently.